pkg/utils/tokens_test.go

```diff
@@ -0,0 +1,102 @@
+package utils
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestComputeTokens(t *testing.T) {
+	required := require.New(t)
+
+	// All the expected data was generated with the original DSE script
+
+	required.Equal(
```

Comment:
We might want to add tests for 2 very common cases: `[]int{3}` and `[]int{3,3}`. 

We could also introduce fuzz testing; in this case, the test could verify e.g. that the resulting ring is evenly distributed. Just a suggestion.